packages/utilities/projects/utilities/src/js/skeleton/getSkeletonStyle.ts

```diff
@@ -0,0 +1,34 @@
+import { borderRadiusMedium, borderRadiusSmall } from '../border';
+import { type Theme, themeLightBackgroundSurface, themeDarkBackgroundSurface } from '../theme';
+
+type BorderRadius = 'small' | 'medium';
+type Options = {
+  borderRadius?: BorderRadius | string;
+  theme?: Theme;
```

Comment:
What about theme auto?

Reply:
Agreed, I think we need to support theme="auto" too in the meantime.

Reply:
That's up to the consumer because we don't know the context and risk potential media query nesting.  
Also it wouldn't match the other style utilities that currently don't support `auto`.